content/applications/websites/livechat/chatbots.rst

```diff
@@ -0,0 +1,240 @@
+========
+Chatbots
+========
+
+A *Chatbot* is a program designed to mimic a conversation with a live human. Chatbots are assigned a
+script of pre-written steps to follow. The scripts are designed to anticipate a visitor's potential
+response, and lead them through a series of questions and answers the same way a live team member
+would.
+
+Chatbots can be customized to fill various roles, from customer support, to creating leads, to
+collecting contact information. The goal of the chatbot depends on the page of the website they are
+assigned, and the messages included in the script, among other criteria.
+
+.. image:: chatbot/chatbot-visitor-view.png
+   :align: center
+   :alt: View of the chat window with a helpdesk ticket created in Odoo Live Chat.
+
+Build a chatbot
+===============
+
+Before creating a new chatbot, the *Live Chat* application must first be installed on the database.
+This can be done directly from the :menuselection:`Apps` menu by searching for `Live Chat` in the
+:guilabel:`search bar` and clicking :guilabel:`Install`.
+
+Alternatively, *Live Chat* can be installed and enabled by navigating to the :menuselection:`Website
+application --> Configuration --> Settings`, and checking the box labeled :guilabel:`Livechat`. Once
+enabled, the database will refresh, and the *Live Chat* application is accessible.
+
+Once the *Live Chat* application has been installed on the database, open it, and go to
+:menuselection:`Configuration --> Chatbots`.
+
+.. note::
+    When the *Live Chat* app is installed, a sample chatbot is created, named *Welcome Bot*. This
+    chatbot has a pre-configured script that walks through a few basic steps, including asking for a
+    visitor's email address, and forwarding the conversation to an operator.
+
+    *Welcome Bot* can be used as a starting point. The existing steps can be edited or removed, and
```

Comment:
I went back and forth on this, however,  in this instance, it is more of an Odoo-specific term since it is a default feature that is pre-loaded when the app is installed. I'd rather keep it italicized so they know I am referring to the Odoo-provided script in this case, and not the general idea of a 'welcome bot'.